### PR TITLE
Add Web Performance to our Guides

### PR DESCRIPTION
**Pull request by sneamonitakis**
I wanted to write this PR to add Web Performance to our Guides. 

This topic has come up a few times on different projects; most recently on True Botanicals' Shopify site while working with @swetagsanghavi. After talking this morning with @tysongach about Web Performance, we noticed it was not covered in the Guides and figured it would be an excellent addition. The ultimate goal could be to evolve this discussion into a service that we could offer; such as Performance Auditing and Improvement.

The other reason I wanted to get this started is because of my limited knowledge of Web Performance. I figured this could be a great way to unearth some internal knowledge we have an as an organization. Are there tools or processes you love? Any learnings you'd like to share?

I am excited to see where this conversation leads.

### CODE REVIEW

**Comment by iwz** [comment]
Yeah!

File: web-performance/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Web Performance
+===============
+
+Web performance refers to the speed in which web pages are downloaded and displayed on the user's web browser. Web performance optimization (WPO), or website optimization is the field of knowledge about increasing web performance.
```
Comment (gylaz) [inline web-performance/README.md]: Worth splitting lines up at 80 chars limit? Will make it easier in the future to see diff changes to text.

Comment (sneamonitakis) [reply web-performance/README.md]: This is a great point. I will edit accordingly, thank you!

File: web-performance/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Web Performance
+===============
+
+Web performance refers to the speed in which web pages are downloaded and
+displayed on the user's web browser. Web performance optimization (WPO), or
```
Comment (mike-burns) [inline web-performance/README.md]:
```suggestion
displayed on the user's web browser. Web performance optimization (WPO) or
```

File: web-performance/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Web Performance
+===============
```
Comment (tysongach) [inline web-performance/README.md]: I think we tend to use the atx-style Markdown headings (using hashes) these days:

```
# Web Performance
```

The setext-style (underlines) limits us to only two heading levels.

File: web-performance/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Web performance refers to the speed in which web pages are downloaded and
+displayed on the user's web browser. Web performance optimization (WPO) or
+website optimization is the field of knowledge about increasing web performance.
+
+
```
Comment (tysongach) [inline web-performance/README.md]: ✂️ extra empty line

File: web-performance/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+* [Demystifying Speed Tooling (Google I/O ’19)](https://www.youtube.com/watch?v=mLjxXPHuIJo)
+* [Fast load times](https://web.dev/fast)
+* [Your first performance budget with Lighthouse](https://bitsofco.de/your-first-performance-budget-with-lighthouse/)
+* [Visualise performance impact between deploys (with Calibre)](https://calibreapp.com/blog/visualise-performance-impact-between-deploys/)
+* [CSS Wizardy Archive](https://csswizardry.com/archive/) - A reverse-
+  chronological list of over 200 posts from CSS Wizardry covering CSS,
+  architecture, performance, and web development.
+
+
```
Comment (tysongach) [inline web-performance/README.md]: ✂️ extra empty line

File: web-performance/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+displayed on the user's web browser. Web performance optimization (WPO) or
+website optimization is the field of knowledge about increasing web performance.
+
+
+Resources
+---------
+
+* [Demystifying Speed Tooling (Google I/O ’19)](https://www.youtube.com/watch?v=mLjxXPHuIJo)
+* [Fast load times](https://web.dev/fast)
+* [Your first performance budget with Lighthouse](https://bitsofco.de/your-first-performance-budget-with-lighthouse/)
```
Comment (tysongach) [inline web-performance/README.md]: Thoughts on using reference-style links, which can help to keep the raw content more parse-able by placing long URLs elsewhere in the document:

```
* [Demystifying Speed Tooling (Google I/O ’19)][google-io-video]
* [Fast load times][web-dev-fast]
* [Your first performance budget with Lighthouse][lighthouse-budgets]

[google-io-video]: https://www.youtube.com/watch?v=mLjxXPHuIJo
[web-dev-fast]: https://web.dev/fast
[lighthouse-budgets]: https://bitsofco.de/your-first-performance-budget-with-lighthouse/
```

File: web-performance/README.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+* [Visualise performance impact between deploys (with Calibre)](https://calibreapp.com/blog/visualise-performance-impact-between-deploys/)
+* [CSS Wizardy Archive](https://csswizardry.com/archive/) - A reverse-
+  chronological list of over 200 posts from CSS Wizardry covering CSS,
+  architecture, performance, and web development.
```
Comment (tysongach) [inline web-performance/README.md]: Thoughts on simplifying this?

```
* [Performance articles from Harry Roberts of CSS Wizardy](https://csswizardry.com/archive/)
```

File: web-performance/README.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+Tools
+-----
+
+* [Google Lighthouse](https://developers.google.com/web/tools/lighthouse/) -
+  Lighthouse is an open-source, automated tool for improving the quality of web
+  pages.
+* [Calibre](https://calibreapp.com/) - World-class web performance monitoring
+  for teams; powered by Google Lighthouse.
```
Comment (tysongach) [inline web-performance/README.md]: “World-class” feels like marketing jargon for Calibre - thoughts on reframing this as to its value to us?

```
* [Calibre](https://calibreapp.com/) - Automated performance monitoring, with
  device emulation, metric budgets, and alerts (powered by Lighthouse)
```

File: web-performance/README.md (hunk 6 above)
Comment (sneamonitakis) [reply web-performance/README.md]: I love this! Didn't know this was an option either. Thanks for sharing 😎

File: web-performance/README.md (hunk 8 above)
Comment (sneamonitakis) [reply web-performance/README.md]: Totally agree. But I have to say, your opinion here is world-class.

**Pull request merged**